tests/e2e/utilities/setup-and-go.ts

```diff
@@ -20,46 +20,74 @@ export async function setupAndGo(
     allowGeo?: boolean;
     amUrl?: string;
     clientId?: string;
+    dialogInput?: string;
     email?: string;
     pw?: string;
     realmPath?: string;
     resourceUrl?: string;
+    selector?: string;
     scope?: string;
     tokenStore?: string;
     tree?: string;
     un?: string;
   },
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-): Promise<{ browser: any; page: any }> {
+): Promise<{ browser: any; messageArray: string[]; networkArray: string[] }> {
   const browser = await browsers[browserType].launch({ headless: true });
-  const context = await browser.newContext({
-    bypassCSP: true,
-    geolocation: { latitude: 24.9884, longitude: -87.3459 },
-    ignoreHTTPSErrors: true,
-    permissions: config && config.allowGeo ? ['geolocation'] : [],
-  });
-  const page = await context.newPage();
+  const messageArray = [];
+  const networkArray = [];
```

Comment:
These changes centralize all event handling, array's onto which we push messages and our selector waits into the same setup function.